src/test/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilderTest.java

```diff
@@ -30,24 +30,30 @@
 @ExtendWith(MockitoExtension.class)
 @Tag(Unit.TAG)
 class CriteriaQueryBuilderTest {
-  CriteriaQueryBuilder criteriaQueryBuilder;
+  private CriteriaQueryBuilder criteriaQueryBuilder;
 
   @BeforeEach
   void setup() {
     criteriaQueryBuilder =
         new CriteriaQueryBuilder(Person.TABLE_NAME, SnapshotBuilderTestData.SETTINGS);
   }
 
+  public static String stripSpaces(String toBeStripped) {
+    return toBeStripped.replaceAll("\\s+", " ").trim();
+  }
+
+  public static void assertQueryEquals(String expectedSql, String actualSql) {
+    assertEquals(stripSpaces(expectedSql), stripSpaces(actualSql), "The sql generated is correct");
```

Comment:
Do you think we want to have the ability to pass in a different assert test string? Or do you expect it always to be `"The sql generated is correct"`

Reply:
Given all the current cases, the text seems OK to me. If we decided we need a different message in the future, we can add an overload to handle that.

Personally I don't find the assert text useful in this case so I'd also be up for removing it completely.